components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css

```diff
@@ -110,7 +110,6 @@
 }
 .xdialog-box .question {
```

Comment:
The whole ruleset can actually be removed. It's supposed to be applied to one dialog only: from the _browse all patients_ table, select _Push to PhenomeCentral_, then _Next: Select patients_, then _Push to PhenomeCentral_. There should be a dialog asking to confirm that the user wants to push the following patients. The padding override doesn't really change much, and the whitespace doesn't need to be preserved. since all that's displayed is a list of patient identifiers.